Libraries/LibWeb/Loader/ResourceLoader.cpp

```diff
@@ -273,17 +273,21 @@ void ResourceLoader::load(LoadRequest& request, GC::Root<SuccessCallback> succes
         Requests::RequestTimingInfo fixme_implement_timing_info {};
 
         // About version page
-        if (url.path_segment_at_index(0) == "version") {
+        if (url.serialize_path() == "version") {
```

Comment:
We call serialize_path() a second time a couple of lines down, can we save that value instead?

Reply:
Good point, done